product/dialogs/miq_dialogs/miq_provision_azure_dialogs_template.yaml

```diff
@@ -0,0 +1,397 @@
+          :data_type: :string
+        :owner_manager: 
+          :description: Name
+          :required: false
+          :display: :edit
+          :data_type: :string
+        :owner_address: 
+          :description: Address
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_company: 
+          :description: Company
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_last_name: 
+          :description: Last Name
+          :required: true
+          :display: :edit
+          :data_type: :string
+        :owner_manager_mail: 
+          :description: E-Mail
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_city: 
+          :description: City
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_department: 
+          :description: Department
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_load_ldap: 
+          :pressed: 
+            :method: :retrieve_ldap
+          :description: Look Up LDAP Email
+          :required: false
+          :display: :show
+          :data_type: :button
+        :owner_manager_phone: 
+          :description: Phone
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_state: 
+          :description: State
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_office: 
+          :description: Office
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_zip: 
+          :description: Zip code
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_email: 
+          :description: E-Mail
+          :required: true
+          :display: :edit
+          :data_type: :string
+        :request_notes:
+          :description: Notes
+          :required: false
+          :display: :edit
+          :data_type: :string
+      :display: :show
+      :field_order: 
+    :purpose: 
+      :description: Purpose
+      :fields: 
+        :vm_tags: 
+          :required_method: :validate_tags
+          :description: Tags
+          :required: false
+          :options: 
+            :include: []
+
+            :order: []
+
+            :single_select: []
+
+            :exclude: []
+
+          :display: :edit
+          :required_tags: []
+
+          :data_type: :integer
+      :display: :show
+      :field_order: 
+    :environment: 
+      :description: Environment
+      :fields:
+        :placement_auto:
+          :values:
+            false: 0
+            true: 1
+          :description: Choose Automatically
+          :required: false
+          :display: :hide
+          :default: false
+          :data_type: :boolean
+        :placement_availability_zone:
```

Comment:
If fields do not apply to Azure they should be removed from the dialog, not hidden.  In the Environment tab this would also include :security_groups and :floating_ip_address.

Excludes :placement_auto since that will be enabled soon.